doc/developer/mzcompose.md

```diff
@@ -203,3 +203,13 @@ def workflow_simple_test(c: Composition):
    w.run("testdrive", "*.td")
    ...
 ```
+
+## Debugging
+
+Here are a few tips for debugging failing Testdrive files:
+
+### Inspecting materialize after a run
+When running a Testdrive mzcompose, upon failure, _usually_ materialize is left running. You can connect with a normal `psql` shell using:
+```
```

Comment:
Why “usually”? Isn’t it “always”?

Also we should recommend `mzcompose sql` rather than `exec`!

Reply:
does `mzcompose sql` give you a shell?

Reply:
Yes!